src/workloads/execution/MultiDeletesBatchedWithSecondaryIndexes_10MB.yml

```diff
@@ -0,0 +1,40 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: >
+  Deletes a range of documents using the BATCHED_DELETE query exec stage while performing writes on another collection.
+  Secondary indexes are created to see how they might affect performance. Documents are approximately of size ~10MB.
+
+Keywords:
+- RunCommand
+- Loader
+- LoggingActor
+- CrudActor
+- insert
+- delete
+- batch
+- deleteMany
+- latency
+- secondary indexes
+
+# TODO move the Clients section to src/phases/execution/MultiDeletes.yml once TIG-3950 is resolved.
```

Comment:
at one point we had a TODO filter, I'm not sure if it will pick this up, typically I've seen it find TODO <ticketnumber>. I'm not sure if it works on this repo.

Might be worth looking into before pushing or making the change so this doesn't go unnoticed later on

Reply:
`git grep TODO src/` doesn't show a consistent comment format for TODO's. @dalyd does genny have an automatic TODO scraper similar to what the MongoDB server has? Does genny recommend a format for TODO messages, or is this one fine?

Reply:
I don't know. That's a good question for #performance-tooling-users and possibly Ryan Timmons.